csrc/device_lower/analysis/tensor_memory.cpp

```diff
@@ -183,6 +188,180 @@ TMemAlllocationInfo computeTMemAlllocationInfo(Fusion* fusion) {
   return result;
 }
 
+// Get the TID Parallel types that we are interested in. We are not interested
+// in the parallel types that are not used in the kernel, and the ones that have
+// size 1. The order of the returned parallel types is from z to x.
+std::vector<ParallelType> getInterestedThreadParallelTypes(Fusion* fusion) {
```

Comment:
nitpick: This function name
```suggestion
std::vector<ParallelType> getNonTrivialActiveThreadParallelTypes(Fusion* fusion) {
```

Reply:
changed